backend/src/model/entity.rs

```diff
@@ -20,214 +20,583 @@ use super::r#enum::{
     water_requirement::WaterRequirement,
 };
 
-/// The `Plants` entity.
-#[allow(clippy::missing_docs_in_private_items)] // TODO: See #97.
+/// The `Plants` entity builds up an hierarchical structure, see `/doc/database/hierarchy.md`:
+///
+#[doc = include_str!("../../../doc/database/hierarchy.md")]
+///
 #[derive(Identifiable, Queryable)]
 #[diesel(table_name = plants)]
 pub struct Plants {
-    /// The internal id of the plant.
+    /// - The internal id of the plant.
+    /// - *Fill ratio:* 100%
     pub id: i32,
-    /// The unique name of the plant. The structure is described in the /doc/database/hierarchy.md document in the repository.
+
+    /// - The unique name of the plant.
+    /// - The structure is described above (`doc/database/hierarchy.md`).
+    /// - *Fill ratio:* 100%
     pub unique_name: String,
-    /// The list of the common names of the plant.
+
+    /// - The list of the common names of the plant in English.
+    /// - *Fetched from* PracticalPlants and Permapeople.
+    /// - *Fill ratio:* 90%
     pub common_name_en: Option<Vec<Option<String>>>,
-    /// The list of the common names of the plant in German. Fetched from Wikidata API if not present in the source datasets.
+
+    /// - The list of the common names of the plant in German.
+    /// - *Fetched from* Wikidata API if not present in any source datasets.
+    /// - *Fill ratio:* 25%
     pub common_name_de: Option<Vec<Option<String>>>,
-    /// Family of the plant.
+
+    /// - Family of the plant.
+    /// - See `/doc/architecture/glossary.md`.
+    /// - *Used* to build up hierarchy.
+    /// - *Fetched from* PracticalPlants and Permapeople.
+    /// - *Fill ratio:* 88%
     pub family: Option<String>,
-    /// Genus of the plant.
+
+    /// - Genus of the plant.
+    /// - Is also first word of unique_name, see above (`/doc/database/hierarchy.md`)
+    /// - *See also* `/doc/architecture/glossary.md`.
+    /// - *Used* to build up hierarchy.
+    /// - *Fetched from* PracticalPlants and Permapeople.
+    /// - *TODO:* copy from first word of unique name.
+    /// - *Fill ratio:* 63%
```

Comment:
Genus is determined by botanical nomenclature. Sometimes gets changed due to modern insights through genetical research.